%YAML:1.0
---
cam_mat: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 6.3718988226523311e+02, 0., 6.3987750928155037e+02, 0.,
       6.3706005836390500e+02, 3.6698200562252384e+02, 0., 0., 1. ]
distort: !!opencv-matrix
   rows: 1
   cols: 5
   dt: d
   data: [ -5.2502931360447945e-02, 5.6306698769444496e-02,
       1.5608055486672912e-04, 8.2284633065163090e-05,
       -1.6525468576360992e-02 ]
